### PR TITLE
Bultek.SPMInstaller version 1.0

### PR DESCRIPTION
**Pull request by mrquantumoff**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/44152)

### CODE REVIEW

File: manifests/b/Bultek/SPMInstaller/1.0/Bultek.SPMInstaller.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Bultek.SPMInstaller
+PackageVersion: 1.0
```
Comment (ItzLevvie) [inline manifests/b/Bultek/SPMInstaller/1.0/Bultek.SPMInstaller.installer.yaml:5]:
```suggestion
PackageVersion: 1.3
```
to follow GitHub Releases

File: manifests/b/Bultek/SPMInstaller/1.0/Bultek.SPMInstaller.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Bultek.SPMInstaller
+PackageVersion: 1.0
```
Comment (ItzLevvie) [inline manifests/b/Bultek/SPMInstaller/1.0/Bultek.SPMInstaller.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.3
```
to follow GitHub Releases

File: manifests/b/Bultek/SPMInstaller/1.0/Bultek.SPMInstaller.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Bultek.SPMInstaller
+PackageVersion: 1.0
```
Comment (ItzLevvie) [inline manifests/b/Bultek/SPMInstaller/1.0/Bultek.SPMInstaller.yaml:5]:
```suggestion
PackageVersion: 1.3
```
to follow GitHub Releases

File: manifests/b/Bultek/SPMInstaller/1.0/Bultek.SPMInstaller.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Bultek.SPMInstaller
+PackageVersion: 1.0
+PackageLocale: en-US
+Publisher: Bultek
+PackageName: SharpPackageManager
```
Comment (ItzLevvie) [inline manifests/b/Bultek/SPMInstaller/1.0/Bultek.SPMInstaller.locale.en-US.yaml:8]:
```suggestion
PackageName: Sharp Package Manager
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by mrquantumoff — COMMENTED** [review]
Even though the app name is SPMInstaller it is an installer, so it should think about the app's version, not the installer's

**Review by mrquantumoff — COMMENTED** [review]
Ok, I'll accept those changes

**Pull request closed without merging**